.github/workflows/erlang-tests.yml

```
@@ -0,0 +1,148 @@
+name: Erlang CI
+
+on:
+  pull_request:
+  push:
+    branches: [ main ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)